editor-resources/github-tutorials/github-tutorial.txt

```diff
@@ -259,6 +259,7 @@ __**Step 8: That's it!**__
 <@212599080101085186>
 <@298636470988832778>
 <@161545097597550593>
+<195662572064604160>
```

Comment:
This is incorrect, please fix and re-commit, please make sure the format is the following

```
<@USER_ID_HERE>
```

Reply:
Fixed in ab33fec6143963f13e826e969b32b322147a40b8